internal/app/certificate_provider_store.go

```diff
@@ -87,3 +88,11 @@ func (s *certificateProviderStore) Put(ctx context.Context, certificateProvider
 	certificateProvider.UpdatedAt = s.now()
 	return s.dynamoClient.Put(ctx, certificateProvider)
 }
+
+func (s *certificateProviderStore) Delete(ctx context.Context, certificateProvider *actor.CertificateProviderProvidedDetails) error {
+	if err := s.dynamoClient.DeleteOne(ctx, certificateProvider.PK, certificateProvider.SK); err != nil {
```

Comment:
this could work out PK and SK from the session, then you wouldn't need to do a `certificateProviderStore.Get` first

Reply:
Ah yeah good point 👍